src/Annotation/ApiResource.php

```diff
@@ -158,6 +160,13 @@ final class ApiResource
      */
     public $shortName;
 
+    /**
+     * @see https://github.com/Haehnchen/idea-php-annotation-plugin/issues/112
+     *
+     * @var bool|null
```

Comment:
```suggestion
     * @var bool
```

It cannot be set to `null` here. `null` is only used as the default value for the attribute.